lms/djangoapps/verify_student/views.py

```diff
@@ -1247,3 +1246,85 @@ def get(self, request, receipt_id):
 
         log.warning(u"Could not find verification with receipt ID %s.", receipt_id)
         raise Http404
+
+
+class UserReceiptIDsView(APIView):
+    """
+    Endpoint for retrieving all receipt IDs associated with a user
+    """
+
+    @method_decorator(require_global_staff)
+    def get(self, request):
+        """
+        GET /verify_student/receipt_ids
+        Returns:
+            200 OK
+            [
+                "xxxxxxxx",
+                "yyyyyyyy"
+            ]
+        """
+        verifications = SoftwareSecurePhotoVerification.get_all_attempts(request.user)
+        if verifications:
+            return Response(verifications)
+        raise Http404
+
+
+class DecryptFaceImageView(APIView):
+    """
+    Endpoint to retrieve decrypted IDV face image data. Can only be used on stage.
+    """
+
+    @method_decorator(require_global_staff)
+    def get(self, request, receipt_id):
+        """
+        Endpoint used for decrypting images on stage based on a given receipt ID
+        GET /verify_student/decrypt-idv-images/face/{receipt_id}
+
+        Returns:
+            200 OK
+            {
+                img
+            }
+        """
+        # if this endpoint is not being accessed on stage, raise a 403. Only stage will have an RSA_PRIVATE_KEY
+        if not settings.VERIFY_STUDENT["SOFTWARE_SECURE"].get("RSA_PRIVATE_KEY", None):
+            return HttpResponseForbidden()
```

Comment:
Can we log a warning here to let our splunk log know someone requested a descryption without the proper RSA private key (presumably someone did this in production)?